src/components/basic/DlWidget/DlWidget.vue

```diff
@@ -7,24 +7,22 @@
             :id="uuid"
             ref="widget"
             :class="widgetStyles"
+            @mouseenter="handleVisibleDragIcon(true)"
+            @mouseleave="handleVisibleDragIcon(false)"
         >
-            <div
-                class="dl-widget__header"
-                @mouseenter="visibleDragIcon = true"
-                @mouseleave="visibleDragIcon = false"
-            >
+            <div class="dl-widget__header">
                 <div class="dl-widget__header--titles">
                     <slot name="header" />
                 </div>
                 <dl-icon
                     :style="`visibility: ${
-                        visibleDragIcon ? 'visible' : 'hidden'
+                        visibleDragIcon && !isDragging ? 'visible' : 'hidden'
                     }`"
                     class="dl-widget__header--drag-icon"
                     icon="icon-dl-drag"
                     color="dl-color-medium"
                     size="15px"
-                    @mousedown="startDragging"
+                    @mousedown.native="startDragging"
```

Comment:
@prodan-eric why do we need .native here? we've used `@mousedown` on dlchip before with no issue and we didnt need to use native. if not having native causes issues please add emitting of the mouse events from the root container in the dl-chip itself and try to not use the deprecated `.native` modifier.

Reply:
I removed it